package.json

```diff
@@ -69,6 +69,9 @@
     "qunit-dom": "^2.0.0",
     "webpack": "^5.74.0"
   },
+  "resolutions": {
+    "ansi-html": "^0.0.9"
```

Comment:
can this resolution be removed? I note that it was originally `^0.0.7` in the yarn.lock, so if we remove this again, yarn lock should remain at `^0.0.9`.
Can any of the other resolutions get the same treatment?

Reply:
Unfortunately `0.0.`7 has a [vulnerability](https://github.com/advisories/GHSA-whgm-jr23-g3j9), that's patched in in `0.0.8`. I bumped to `0.0.9` because it's one bigger and the code difference is negligible.